extensions/amp-story/1.0/amp-story-interactive-poll.css

```diff
@@ -0,0 +1,126 @@
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.i-amphtml-story-interactive-poll-container {
+  background-color: var(--interactive-prompt-background-color) !important;
+}
+
+.i-amphtml-story-interactive-option-container {
+  background-color: var(--i-amphtml-interactive-option-background-color) !important;
+  border-radius: 16px !important;
+  color: var(--i-amphtml-interactive-strong-text-color) !important;
+  font-weight: bold !important;
+  padding: 8px !important;
+}
+
+.i-amphtml-story-interactive-option {
+  display: flex !important;
+  align-items: center !important;
+  justify-content: space-between !important;
+  text-align: center !important;
+  flex-direction: row !important;
+  height: 58px !important;
+  box-sizing: border-box !important;
+  overflow: hidden !important;
+  padding: 0px 12px !important;
+  position: relative !important;
+  box-shadow: var(--i-amphtml-interactive-chip-shadow), inset 0px 0px 0px 1px var(--i-amphtml-interactive-theme-border) !important;
+  -webkit-box-shadow: var(--i-amphtml-interactive-chip-shadow), inset 0px 0px 0px 1px var(--i-amphtml-interactive-theme-border) !important;
+  -moz-box-shadow: var(--i-amphtml-interactive-chip-shadow), inset 0px 0px 0px 1px var(--i-amphtml-interactive-theme-border) !important;
+  border-radius: 8px !important;
+}
+
+.i-amphtml-story-interactive-option:not(:first-child) {
+  margin-top: 8px !important;
+}
+
+.i-amphtml-story-interactive-option.i-amphtml-story-interactive-option-selected {
+  background-color: var(--i-amphtml-interactive-theme-shading);
+  box-shadow: var(--i-amphtml-interactive-chip-shadow-inset), inset 0px 0px 0px 1px var(--i-amphtml-interactive-theme-border) !important;
+  -webkit-box-shadow: var(--i-amphtml-interactive-chip-shadow-inset), inset 0px 0px 0px 1px var(--i-amphtml-interactive-theme-border) !important;
+  -moz-box-shadow: var(--i-amphtml-interactive-chip-shadow-inset), inset 0px 0px 0px 1px var(--i-amphtml-interactive-theme-border) !important;
+  animation: flashBGColor forwards var(--i-amphtml-interactive-animation-time) var(--i-amphtml-interactive-ease-out-curve) !important;
+}
+
+.i-amphtml-story-interactive-option::after {
+  content: '' !important;
+  background-color: var(--i-amphtml-interactive-theme-shading) !important;
+  position: absolute !important;
+  height: 100% !important;
+  width: 100% !important;
+  left: 0 !important;
+  top: 0 !important;
+  transform: translateX(-200%) !important;
```

Comment:
Why not `-100%` ?

Reply:
Good catch, not sure why I did 200... Updated